src/utils/logWithTime.js

```diff
@@ -15,6 +14,9 @@ const LogWithTime = (message, statesArray) => {
 	}
 	console.log('');
```

Comment:
Be careful with hanging console logs.
It would be great if you can use https://eslint.org/ _(but take it as a separate thing)_